website/content/plugins/drivers/podman.mdx

```diff
@@ -161,6 +170,15 @@ The `podman` driver implements the following [capabilities](/nomad/docs/concepts
   }
   ```
 
+- `auth_soft_fail` - (Optional) Ignores errors returned from auth backend so
+  fallbacks can be tried.
```

Comment:
```suggestion
- `auth_soft_fail` - (Optional) Ignore errors returned from auth backend so Podman can fall back to a different auth method.
```

We are trying to eliminate passive voice (fallbacks can be tried) and use active voice instead. I'm not really sure about Podman so guessed at the functionality based on some searching.